static/app/views/dashboards/widgets/timeSeriesWidget/plottables/area.tsx

```diff
@@ -29,9 +29,8 @@ export class Area extends AggregateTimeSeries<AreaConfig> implements PlottableDa
   toSeries(plottingOptions: AggregateTimePlottingOptions): LineSeriesOption[] {
     const {timeSeries, config = {}} = this;
 
-    const {color, unit} = plottingOptions;
-
-    const scaledSeries = this.scaleToUnit(unit);
+    const color = plottingOptions.color ?? config.color ?? undefined;
```

Comment:
Nit, you can probably return the `undefined` default as the nullish coalescing handles that. Unless you want to specifically avoid the value being set to null
```suggestion
    const color = plottingOptions.color ?? config.color;
```

Reply:
I actually just realized that `plottingOptions.color` is guaranteed to be set anyway, so I can remove that piece!

Reply:
(I'll do that in a follow-up PR)